lib/awesome_print/formatters/base_formatter.rb

```diff
@@ -123,18 +123,29 @@ def outdent
 
       def align(value, width)
         if options[:multiline]
-          if options[:indent] > 0
-            value.rjust(width)
-          elsif options[:indent] == 0
-            indent + value.ljust(width)
+          if options[:justify_hash]
+            align_justify(value, width)
           else
-            indent[0, indentation + options[:indent]] + value.ljust(width)
+            align_no_justify(value)
           end
         else
           value
         end
       end
 
+      def align_justify(value, width) # rubocop:disable Metrics/AbcSize
```

Comment:
Please remove any rubocop disable comments; They're a bad practice because it'll never report for this metric again.